tests/integration/telemetry/stats/prometheus/footprint/footprint_test.go

```diff
@@ -0,0 +1,474 @@
+		"envoy_server_concurrency":                                                     true,
+		"envoy_server_days_until_first_cert_expiring":                                  true,
+		"envoy_server_debug_assertion_failures":                                        true,
+		"envoy_server_dropped_stat_flushes":                                            true,
+		"envoy_server_dynamic_unknown_fields":                                          true,
+		"envoy_server_envoy_bug_failures":                                              true,
+		"envoy_server_hot_restart_epoch":                                               true,
+		"envoy_server_initialization_time_ms_bucket":                                   true,
+		"envoy_server_initialization_time_ms_count":                                    true,
+		"envoy_server_initialization_time_ms_sum":                                      true,
+		"envoy_server_live":                                                            true,
+		"envoy_server_main_thread_watchdog_mega_miss":                                  true,
+		"envoy_server_main_thread_watchdog_miss":                                       true,
+		"envoy_server_memory_allocated":                                                true,
+		"envoy_server_memory_heap_size":                                                true,
+		"envoy_server_memory_physical_size":                                            true,
+		"envoy_server_parent_connections":                                              true,
+		"envoy_server_seconds_until_first_ocsp_response_expiring":                      true,
+		"envoy_server_state":                                                           true,
+		"envoy_server_static_unknown_fields":                                           true,
+		"envoy_server_stats_recent_lookups":                                            true,
+		"envoy_server_total_connections":                                               true,
+		"envoy_server_uptime":                                                          true,
+		"envoy_server_version":                                                         true,
+		"envoy_server_wip_protos":                                                      true,
+		"envoy_server_worker_0_watchdog_mega_miss":                                     true,
+		"envoy_server_worker_0_watchdog_miss":                                          true,
+		"envoy_server_worker_1_watchdog_mega_miss":                                     true,
+		"envoy_server_worker_1_watchdog_miss":                                          true,
+		"envoy_server_worker_2_watchdog_mega_miss":                                     true,
+		"envoy_server_worker_2_watchdog_miss":                                          true,
+		"envoy_server_worker_3_watchdog_mega_miss":                                     true,
+		"envoy_server_worker_3_watchdog_miss":                                          true,
```

Comment:
This seems likely flaky - I think you will get one per core for ingress, so on a !4 core machine it will fail(?)

Side note: is this a bug? seems worker should be a tag...

Reply:
I raised this here: https://github.com/istio/istio/issues/20235#issuecomment-971538339 because I think it's a bug too

Reply:
The actual underlying problem is an Envoy bug-turned-feature, iirc, because it wasn't caught for at least one release.

But yes, this test is also probably flaky, and needs to be fixed.